src/name/bizna/jarm/BusErrorException.java

```diff
@@ -2,4 +2,8 @@
 
 public final class BusErrorException extends Exception {
 	static final long serialVersionUID = 1;
+
+	public BusErrorException(String message) {
+		super(message);
+	}
```

Comment:
Sorry it took me so long to review this one. I like the new one. It's a little less tolerant of border cases than I'd like (such as a null `reason`) but it'll be fine until (unless?) "real exceptions" are added.